### ISSUE TITLE
Missing MUDetect-1 for project chensun cf23b99

### ISSUE THREAD

**Issue by lafenzo** [stg-tud/MUBench#441]
Missing MUDetect-1 for project chensun cf23b99

**Comment by salsolatragus** [comment]
Could you say a little more? When and where is it missing? What does missing mean here? How did you come to this conclusion? How can I reproduce it? What do you suggest?

**Comment by lafenzo** [comment]
Missing information(api, description, location...) about misuse MUDetect-1(which was found in paper MSR'19 in project chensun, in line 37 of http://www.st.informatik.tu-darmstadt.de/artifacts/mudetect/Experiment%20R.html), I can't find it in this page(https://github.com/stg-tud/MUBench/tree/master/data/chensun/misuses)

**Comment by salsolatragus** [comment]
I deleted MUDetect-1 with ddaab45, because we found that is is actually a false positive. The `PreparedStatement` is correctly closed by [the call to `DBManager.closeStatement` on line 226](http://www.st.informatik.tu-darmstadt.de/artifacts/mubench/reviews/fse18/experiments/2/detectors/MuDetect/projects/chensun/versions/cf23b99/misuses/finding-1/reviewers/reviewer1.html), even though the API documentation of `DBManager` says it should be closed using `DBManager.closePreparedStatement` (both methods are implemented functionally equivalent).

Consequently, the misuse should've been removed from the experiments, which I apparently failed to do. I'm sorry for this confusion.
